### PR TITLE
Feature/spack flux

### PR DESCRIPTION
**Pull request by christopherwharrop-noaa**
This PR completes the process of removing the use of conda from Chiltepin.  An installation script to install and build flux and pytest and parsl is added for use on HPC machines.  Also, test configurations for Hera and Hercules are updated to use the official spack-stack installations on those machines.

### CODE REVIEW

**Comment by christopherwharrop-noaa** [comment]
@NaureenBharwaniNOAA - This is ready for you.

File: README.md [hunk 1]
```diff
@@ -33,26 +33,57 @@ because the Flux scheduler does not support MacOS.  It can be used, however,
...
-conda env create --name chiltepin --file chiltepin.yml
+cd install
+./install.sh
 ```
 
-## Building and running Chiltepin container
+## Activate the Chiltepin spack environment
+
+After the chiltepin spack environment is built it must be activated. While
+the installation step above need only be done once, this step must be done
```
Comment (NaureenBharwaniNOAA) [inline README.md:74]: 'needs to only be done once`

File: install/install.sh [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+spack mirror list
+
+# Add flux and pytest to chiltepin environment
+spack add python
+spack add py-pip
+spack add py-pytest
+spack add flux-core@0.53.0
+spack add flux-sched@0.28.0
+
+# Concretize and install
```
Comment (NaureenBharwaniNOAA) [inline install/install.sh:35]: `install ___`? The spack?

**Review by NaureenBharwaniNOAA — DISMISSED** [review]
Just a couple small possible edits.

File: install/install.sh (hunk 2 above)
Comment (christopherwharrop-noaa) [reply install/install.sh]: Thanks for the feedback.  I've pushed some updates.  I'll need you to re-approve if you like the changes.

**Review by NaureenBharwaniNOAA — APPROVED** [review]
Once it passes, approval looks good to me!

**Pull request merged**